Dfe.Academies.External.Web/Pages/Trust/JoinAMat/TrustConsent.cshtml.cs

```diff
@@ -13,7 +13,7 @@ public class TrustConsent : BaseSchoolPageEditModel
 	{
 		private readonly IFileUploadService _fileUploadService;
 		
-		public TrustConsent(IConversionApplicationRetrievalService conversionApplicationRetrievalService, IReferenceDataRetrievalService referenceDataRetrievalService, IConversionApplicationCreationService conversionApplicationCreationService, IFileUploadService fileUploadService) : base(conversionApplicationRetrievalService, referenceDataRetrievalService, conversionApplicationCreationService, "ApplicationSchoolJoinAMatTrustSummary")
+		public TrustConsent(IConversionApplicationRetrievalService conversionApplicationRetrievalService, IReferenceDataRetrievalService referenceDataRetrievalService, IConversionApplicationCreationService conversionApplicationCreationService, IFileUploadService fileUploadService) : base(conversionApplicationRetrievalService, referenceDataRetrievalService, conversionApplicationCreationService, "ApplicationSchoolChangesToATrust")
```

Comment:
ApplicationId && Urn property can be removed as in base class

Reply:
done